chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1168,7 +1160,13 @@ impl PeerManagerActor {
     ///  - bootstrap outbound connections from known peers,
     ///  - unban peers that have been banned for awhile,
     ///  - remove expired peers,
-    fn monitor_peers_trigger(&mut self, ctx: &mut Context<Self>, max_interval: Duration) {
+    fn monitor_peers_trigger(
+        &mut self,
+        ctx: &mut Context<Self>,
+        mut interval: Duration,
+        default_interval: Duration,
+        max_interval: Duration,
```

Comment:
optional:
we have 3 Duration types next to each other.. 
Maybe we should pass 'default_interval' and 'max_interval' as a tuple?

Reply:
@mm-near I make it into a tuple. It looks better now.